sdk/translation/ai-translation-text/samples/v1-beta/typescript/src/breakSentence.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary This sample demonstrates how to make a simple call to the Azure Text Translator service to get sentences' boundaries.
+ */
+import TextTranslationFactory, { ErrorResponseOutput, TranslatorCredential, InputTextItem, FindSentenceBoundariesQueryParamProperties, BreakSentenceItemOutput } from "@azure-rest/ai-translation-text";
```

Comment:
It is a bit unusual to call the client a factory, could we just call it `TextTranslationClient`?

Reply:
Changed to `TextTranslationClient`.